pkg/providers/v1/aws_fakes.go

```diff
@@ -69,7 +69,7 @@ func NewFakeAWSServices(clusterID string) *FakeAWSServices {
 	selfInstance := &ec2.Instance{}
 	selfInstance.InstanceId = aws.String("i-self")
 	selfInstance.Placement = &ec2.Placement{
-		AvailabilityZone: aws.String("us-east-1a"),
+		AvailabilityZone: aws.String("us-west-2a"),
```

Comment:
The inconsistency of region was causing issues on some tests, so making the region consistent here.